### ISSUE TITLE
Add option to move JS and CSS files to differenct directory than the dist folder

### ISSUE THREAD

**Issue by rohankumar1524** [angular/angular-cli#24856]
### Command

build

### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

It might be duplicate of https://github.com/angular/angular-cli/issues/2688. But there is no solution I found to move JS and CSS files to specific folder at the time of build. Even the deploy-url is deprecated. What's the robust solution to do this job without using any external node script to move the files.

### Minimal Reproduction

I am using Angular 14 version for now o test this issue.

### Your Environment

```text
ng build
```

**Comment by alan-agius4** [comment]
Hi, what is the use-case that you need to move the CSS and JS into a different directory?

**Comment by rohankumar1524** [comment]
Yes actually I am working on micro-frontend, the unauthorized like login/register are made with next js and the authorized like user/dashboard, user/profile are made in Angular. So the pointing to assets like js/css/fonts needs to be in a separate folder like dist/user/{*.js|*.css*.fonts) and in index.html the src/href should be updated accordingly.

**Comment by alan-agius4** [comment]
This is rather a specific use-case. In this case it would be better to extend the Angular CLI 3rd party builders. Like https://www.npmjs.com/package/ngx-build-plus
